github/resource_github_team_membership_test.go

```diff
@@ -227,6 +237,12 @@ resource "github_team_membership" "test_team_membership" {
   username = "%s"
   role     = "%s"
 }
+
+resource "github_team_membership" "test_team_membership_slug" {
+  team_id  = "${github_team.test_team.slug}"
+  username = "%s"
+  role     = "%s"
+}
 `, username, randString, username, role)
```

Comment:
Looks like we'll need another argument here on line 246 so that the team slug can be inserted into the above string!